src/Sfa.Das.ApprenticeshipInfoService.Infrastructure/Elasticsearch/ElasticsearchClientFactory.cs

```diff
@@ -16,16 +19,31 @@ public ElasticsearchClientFactory(IConfigurationSettings applicationSettings)
 
         public IElasticClient Create()
         {
-            using (var settings = new ConnectionSettings(new StaticConnectionPool(_applicationSettings.ElasticServerUrls)))
+            ConnectionSettings settings;
```

Comment:
I don't think this IDisposable ConnectionSettings object has an explicit Dispose.